tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/validation/TclValidator.xtend

```diff
@@ -290,6 +294,25 @@ class TclValidator extends AbstractTclValidator {
 			}
 		]
 	}
+	
+	private def Map<String, Set<JvmTypeReference>> collectAssignmentVariables(TestStepContext context){
```

Comment:
The naming of this method confuses me. It returns all declared variables in a given context. I would expect a name such as `getDeclaredVariables`. Also it could reside in `TclModelUtil`